easybuild/easyconfigs/g/GCCcore/GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'EB_GCC'
+
+name = 'GCCcore'
+version = '7.3.0'
+
+homepage = 'http://gcc.gnu.org/'
+description = """The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
+ as well as libraries for these languages (libstdc++, libgcj,...)."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+mpfr_version = '3.1.6'
+
+source_urls = [
+    'http://ftpmirror.gnu.org/gnu/gcc/gcc-%(version)s',  # GCC auto-resolving HTTP mirror
+    'http://ftpmirror.gnu.org/gnu/gmp',  # idem for GMP
+    'http://ftpmirror.gnu.org/gnu/mpfr',  # idem for MPFR
+    'http://ftpmirror.gnu.org/gnu/mpc',  # idem for MPC
+    'ftp://gcc.gnu.org/pub/gcc/infrastructure/',  # GCC dependencies
+    'http://gcc.cybermirror.org/infrastructure/',  # HTTP mirror for GCC dependencies
+    'http://isl.gforge.inria.fr/',  # original HTTP source for ISL
+]
+sources = [
+    'gcc-%(version)s.tar.gz',
+    'gmp-6.1.2.tar.bz2',
+    'mpfr-%s.tar.bz2' % mpfr_version,
+    'mpc-1.0.3.tar.gz',
+    'isl-0.19.tar.bz2',
+]
+patches = [
+    ('mpfr-%s-allpatches-20180412.patch' % mpfr_version, '../mpfr-%s' % mpfr_version),
+    'GCCcore-6.2.0-fix-find-isl.patch',
+]
+checksums = [
+    'fa06e455ca198ddc11ea4ddf2a394cf7cfb66aa7e0ab98cc1184189f1d405870',  # gcc-7.2.0.tar.gz
```

Comment:
@jhein32 Comment still says `7.2.0`, please use `eb --inject-checksums` to correct this (don't do it manually)